metals/src/main/scala/scala/meta/internal/zipkin/ZipkinProperties.scala

```diff
@@ -4,20 +4,20 @@ object ZipkinProperties {
 
   val zipkinServerUrl: Property = Property("metals.zipkin.server.url")
 
-  val debug: Property = Property("metals.bloop.trace.debug")
+  val debug: Property = Property("metals.bloop.tracing.debug")
```

Comment:
Remember debug has changed! It's now `debugTracing` :smile:

Reply:
Good catch!